AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/handler/FragmentArgHandler.java

```diff
@@ -113,5 +113,12 @@ private void createBuilderInjectionMethod(Element element, EFragmentHolder holde
 		JVar arg = method.param(paramClass, fieldName);
 		method.body().invoke(builderArgsField, bundleHelper.getMethodNameToSave()).arg(argKeyStaticField).arg(arg);
 		method.body()._return(_this());
+
+		String docComment = processingEnv.getElementUtils().getDocComment(element);
+		if (docComment != null) {
+			method.javadoc().append(docComment.replaceAll("\r", "").trim());
```

Comment:
Yes i mean that i removed the `\r`, sorry.

the `.getDocComment(element)` call returns comment with `\r\n` but when i apped that to the javadoc i get the line-break without an `*` in the line (like the first snippet). removing the `\r` resulted in having a correctly formatted javadoc output.